src/partials/hero-header.html

```diff
@@ -0,0 +1,51 @@
+<header class="header">
+  <div class="header__logo">
+    <a class="logo" href="#"
+      ><img src="../images/hero-header/logo x1.png" alt="Logo of the company" width="199"
+    /></a>
+  </div>
+
+  <nav class="navigation">
+    <ul class="navigation__list">
+      <li>
+        <a href="./index.html" class="navigation__item">Home</a>
+      </li>
+      <li>
+        <a href="#abaut" class="navigation__item">About us</a>
+      </li>
+      <li>
+        <a href="#howitsmade" class="navigation__item">How it’s made?</a>
+      </li>
+      <li>
+        <a href="#products" class="navigation__item">Our products</a>
+      </li>
+      <li>
+        <a href="#contact" class="navigation__item">Contact</a>
+      </li>
+    </ul>
+  </nav>
+  <div class="button">
+    <button type="button" class="button_order">Buy now ></button>
```

Comment:
jak chcesz tworzyć nazwy klas dwu słowne to użwa się "-" do odzielenia słów np:. "button-order"